flow/model/qvalue/dwh.go

```diff
@@ -10,16 +10,20 @@ import (
 )
 
 func DetermineNumericSettingForDWH(precision int16, scale int16, dwh protos.DBType) (int16, int16) {
-	if dwh == protos.DBType_CLICKHOUSE {
-		if precision > numeric.PeerDBClickhousePrecision || precision <= 0 ||
-			scale > precision || scale < 0 {
-			return numeric.PeerDBClickhousePrecision, numeric.PeerDBClickhouseScale
-		}
-	} else {
-		if precision > numeric.PeerDBNumericPrecision || precision <= 0 ||
-			scale > numeric.PeerDBNumericScale || scale < 0 {
-			return numeric.PeerDBNumericPrecision, numeric.PeerDBNumericScale
-		}
+	var warehouseNumeric numeric.WarehouseNumericComaptibility
+	switch dwh {
+	case protos.DBType_CLICKHOUSE:
+		warehouseNumeric = numeric.ClickHouseNumericCompatibility{}
+	case protos.DBType_SNOWFLAKE:
+		warehouseNumeric = numeric.SnowflakeNumericCompatibility{}
+	case protos.DBType_BIGQUERY:
+		warehouseNumeric = numeric.BigQueryNumericCompatibility{}
+	default:
+		warehouseNumeric = numeric.SnowflakeNumericCompatibility{}
```

Comment:
```suggestion
	default:
		warehouseNumeric = numeric.DefaultNumericCompatibility{}
```
Either implement a default impl that allows whatever, or panic here